devops/ansible/examples/girder-dev-environment/site.yml

```diff
@@ -70,3 +70,11 @@
       become: yes
       become_user: root
       when: ansible_os_family == "RedHat"
+
+    - name: Activate virtualenv on login
+      lineinfile:
+        dest: ~/.profile
+        line: 'source {{girder_virtualenv}}/bin/activate'
```

Comment:
Ansible style dictates that we pad the variable with spaces like: `{{ girder_virtualenv }}`.